test/e2e/velero_utils.go

```diff
@@ -261,6 +258,30 @@ func veleroBackupNamespace(ctx context.Context, veleroCLI string, veleroNamespac
 	return err
 }
 
+// veleroBackupExcludeNamespaces uses the veleroCLI to backup a namespace.
+func veleroBackupExcludeNamespaces(ctx context.Context, veleroCLI string, veleroNamespace string, backupName string, excludeNamespaces []string) error {
+	namespaces := ""
+	for nsNum, curNamespace := range excludeNamespaces {
```

Comment:
Could we use [`strings.Join`](https://golang.org/pkg/strings/#Join) here?

Reply:
Yup, thanks, changed that.